### PR TITLE
Fix Global Config Test

### PR DESCRIPTION
**Pull request by gkillough**
No distinction was being made between success and error messages in validation.

### CODE REVIEW

File: src/main/js/dynamic/GlobalConfiguration.js [hunk 1]
```diff
@@ -6,7 +6,11 @@ import FieldsPanel from 'field/FieldsPanel';
 import ConfigurationLabel from 'component/common/ConfigurationLabel';
```
Comment (gkillough) [inline src/main/js/dynamic/GlobalConfiguration.js:7]: The changes here are all linting. I originally thought I had to modify something here, but did not end up needing to. I decided to leave the linting as an incremental improvement.

File: src/main/js/dynamic/GlobalConfiguration.js [hunk 2]
```diff
@@ -24,37 +28,40 @@ class GlobalConfiguration extends React.Component {
...
             currentFields: fields,
             showTest: false
         };
     }
 
     componentDidMount() {
-        const fieldModel = this.state.currentConfig;
-        this.props.getConfig(fieldModel.descriptorName);
+        const { currentConfig } = this.state;
+        const { getConfig: getConfig1 } = this.props;
```
Comment (bamandel) [inline src/main/js/dynamic/GlobalConfiguration.js:45]: I'm not a fan of this name. Why is it getConfig1? If it's a duplicate item there should be a better name.

File: src/main/js/dynamic/GlobalConfiguration.js [hunk 3]
```diff
@@ -24,37 +28,40 @@ class GlobalConfiguration extends React.Component {
...
+        getConfig1(currentConfig.descriptorName);
     }
 
     componentDidUpdate(prevProps, prevState, snapshot) {
         const { currentFields, currentDescriptor } = this.state;
-        if (this.props.currentConfig !== prevProps.currentConfig && this.props.updateStatus === 'DELETED') {
+        const { updateStatus, updateConfig: updateConfig1, currentConfig } = this.props;
```
Comment (bamandel) [inline src/main/js/dynamic/GlobalConfiguration.js:51]: This has updateConfig1, there should be a better name for this.

**Review by bamandel — APPROVED** [review]

File: src/main/js/dynamic/GlobalConfiguration.js (hunk 1 above)
Comment (bamandel) [reply src/main/js/dynamic/GlobalConfiguration.js:7]: Just saw this message, the linter made some questionable function names that we will probably want to change.

Comment (psantos1113) [reply src/main/js/dynamic/GlobalConfiguration.js:7]: we just have to make sure that that function names mapDispatchToProps section aren't the same as the function names used in the dispatch functions in the same section.  That is the linting error called shadowing if my memory is correct.

**Review by psantos1113 — APPROVED** [review]

**Review by ChomickiM — APPROVED** [review]

**Pull request merged**